### PR TITLE
Add recipe for focaccia

### PR DESCRIPTION
**Pull request by oxo42**

### CODE REVIEW

File: focaccia.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Focaccia
+
+A really nice easy Focaccia.  It's basically [Babish's recipe](https://basicswithbabish.co/basicsepisodes/bread) halved and units changed to weight rather than volume.
+
+## Equipment
+* 9x13" deep walled baking pan
+* Stand mixer with dough hook
+
```
Comment (dgadling) [inline focaccia.md:10]:
```suggestion
* Large bowl, rubber spatula
```

File: focaccia.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Focaccia
+
+A really nice easy Focaccia.  It's basically [Babish's recipe](https://basicswithbabish.co/basicsepisodes/bread) halved and units changed to weight rather than volume.
+
+## Equipment
+* 9x13" deep walled baking pan
```
Comment (dgadling) [inline focaccia.md]:
```suggestion
* 9x13" deep walled baking sheet
```

File: focaccia.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+3. Once water is fully absorbed add 1 Tbsp of kosher salt, and then knead on a medium speed for about 5 minutes.   This is a good point to add in any extras such as olives or rosemary.
+4. Generously oil a bowl, add your dough mixture, cover, and let sit for about 2 hours.
+5. Using a rubber spatula, fold the dough stuck to the edges onto itself.
+6. In an 9x13 rimmed baking sheet, add about ½ a cup of olive oil and spread out until pan is fully coated.
+7. Place the bread mixture onto the baking sheet and using the olive oil, fold the mixture onto itself a few times and then start to spread the dough across the surface of the pan. If the dough keeps springing back, place some olive oil on the bread and cover it with plastic wrap for 15 minutes. You may have to do it a few times, but it’ll eventually spread to the whole pan.
+8. Once the pan is filled out, hit it with some more oil, and then cover and let it ferment overnight in the fridge so for about 12 hours. If any bubbles form, just poke them with a knife, and using oiled fingers, pull the bread just off the edges.
```
Comment (dgadling) [inline focaccia.md]:
```suggestion
8. Once the pan is filled out, hit it with some more oil, and then cover and let it ferment overnight in the fridge (about 12 hours). If any bubbles form, just poke them with a knife, and using oiled fingers, pull the bread just off the edges.
```

File: focaccia.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+9. Coat bread in olive oil and let sit covered at room temperature for about an hour. Make dimples in the dough with your fingers.
+10. Preheat oven at 230°C (450°F) for 1 hour. Coat the dough one last time with olive oil and kosher salt and bake for about 30 minutes.
```
Comment (dgadling) [inline focaccia.md]: Preheat for **an hour**? 😱 Add that to step 9?

**Review by dgadling — APPROVED** [review]
Looks pretty great, thanks for the addition!

File: focaccia.md (hunk 2 above)
Comment (dgadling) [reply focaccia.md]: Just for consistency, use whichever (pan or sheet) is most correct :)

**Comment by oxo42** [comment]
This is ready to merge

**Pull request merged**